howtos/centos_to_opensuse.rst

```diff
@@ -0,0 +1,212 @@
+but will destroy all data on the target disk.
+Hence the advise here to preserve the original system disk and to detach it prior to the new install.
+
+.. warning::
+
+    If, for whatever reason, there is a need to revert to the v3 system disk,
+    avoid having more than one Rockstor system disk, of **any** version, attached simultaneously.
+    This causes known confusion within the current and past versions of the Web-UI.
+    This inflexibility will be improved in time.
+
+.. _disconnect_data_disks:
+
+Disconnect all Data disks
+-------------------------
+
+As a purely precautionary measure, it is highly advisable that all data disks be detached
+prior to the new install.
+Take careful note of their connections to the host system.
+This connection concern relates to potential hardware compatibility of these interconnects.
+Btrfs, our underlying file system and device manager, is not normally concerned with such changes,
+but the existing hardware pairings, assuming prior function, are best noted nevertheless.
+Just in case, after the last shutdown of v3, while attaching the new system disk,
+be sure to unplug all prior disks.
+This is simply to avoid an accidental selection of a data pool member for the fresh install.
+
+.. warning::
+
+    As stated above, the new openSUSE install will wipe all prior data on the target disk selected.
+    A simple quick mistake at the initial :ref:`installer_select_disk` step could inadvertently
+    destroy a pool member's data.
+
+.. _openSUSE_installer:
+
+OpenSUSE Installer
+------------------
+
+See :ref:`installer_howto` for a step-by-step explanation and guide.
+And again, great care should be taken on the early :ref:`installer_select_disk` (intended target system disk)
+choice.
+If the above advise is followed, there will only be one newly attached proposed system disk anyway.
+
+Once the new install is in place, it is advisable to apply all upstream updates.
+See: :ref:`updaterockstorwebui`.
+Take care to ensure these have all been applied prior to rebooting.
+The Dashboard can help to determine this by observing the network and CPU activity.
+*There is an outstanding bug where our 'wifi-like' busy indicator does not last the duration of the installs.*
+
+Ensure that the system does reboot and return as expected before re-attaching all prior pool members,
+connected as before, perform the pool import and then optionally a config restore.
+
+
+.. _openSUSE_import_notes:
+
+OpenSUSE versions Pool/s import
+-------------------------------
+
+Under the OpenSUSE version, the Pool import is performed the same as under v3, initiated via the :ref:`disks` overview page.
+See: :ref:`import_data`.
+
+.. warning::
+
+    This is an upstream decision and **not** enacted by Rockstor.
+    See our :ref:`redundancyprofiles` for more information,
+    and a suggested workaround if needed.
+    See also :ref:`btrfsunwellimport` in case a pool requires special mount options.
```

Comment:
```suggestion
    See also :ref:`btrfsunwellimport` in case a pool requires special mount options.
```

@Hooverdan96 just a little left-over to be removed there. Pertaining to the now removed read-only default on parity raid. I've just removed that reference and left the poorly-pool import as a note.